acceptance/setup/pre_suite/15_setup_repos.rb

```diff
@@ -1,6 +1,14 @@
 def initialize_repo_on_host(host, os, nightly)
   case os
   when :debian
+
+    # For openjdk8
+    if host.platform.start_with?('debian-8')
```

Comment:
I wondered about that (from Jeremy's PR), but didn't know if that'd be available to us too here; in any case, I agree it'd be preferable, I was just (conservatively) matching what we already had.  I'll adjust it.